docs/en_us/deployment.md

```diff
@@ -36,4 +36,31 @@ kubectl apply -f https://raw.githubusercontent.com/bfenetworks/ingress-bfe/devel
 ``` shell script
 kubectl apply -f https://raw.githubusercontent.com/bfenetworks/ingress-bfe/develop/examples/ingress.yaml  
 ```
+* Create ingress resource for testing service to verify the installation if your kubernetes version > 1.19
+
+``` shell script
+kubectl apply -f ingress-gt-1.19.yaml  
```

Comment:
Could you please make following additional change? thx: 
1. commit the code into branch 'develop'
2. separate ingress config into a file, for example，ingress-v1.yaml，and use this file in 'kubectl apply'

Reply:
thanks for your sugesstion!